frontend_build/src/webpack.config.base.js

```diff
@@ -79,8 +79,14 @@ var config = {
           preserveWhitespace: false,
           loaders: {
             js: 'buble-loader',
-            stylus: combineLoaders(vueStylusLoaders),
-            scss: combineLoaders(vueSassLoaders),
+            stylus: ExtractTextPlugin.extract({
+              fallback: 'style-loader',
```

Comment:
Yes - although, you're right, and we should get rid of the fallback, and use `allChunks: true` instead (the default, false causes CSS in extracted chunks to not be bundled into CSS).